core/backend/src/internal/workspace/WorkspaceImpl.ts

```diff
@@ -582,7 +585,7 @@ class EditorContainerImpl extends WorkspaceContainerImpl implements EditableWork
     }
   }
 
-  public async createDb(args: { dbName?: string, version?: string, manifest: WorkspaceDbManifest }): Promise<EditableWorkspaceDb> {
+  public async createDb(args: { dbName?: string, manifest: WorkspaceDbManifest }): Promise<EditableWorkspaceDb> {
```

Comment:
I chose to remove the version arg of createDb.

 - You wouldn't be able to edit that version anyway (if its a released version)
 - I cant think of a valid workflow where a new db wouldn't start at 0.0.0.